jaxopt/_src/zoom_linesearch2.py

```diff
@@ -0,0 +1,612 @@
+# Copyright 2023 Google LLC
```

Comment:
It's ok to remove the old code and put your code instead (no need for a new file).